npm-global-or-local-packages is misleading #370 Fixed (#371)
nodejs/nodejs.dev: src/documentation/0026-npm-packages-local-global/index.md

- **local packages** are installed in the directory where you run `npm install <package-name>`, and they are put in the `node_modules` folder under this directory
- **global packages** are all put in a single place in your system (exactly where depends on your setup), regardless of where you run `npm install -g <package-name>`

> In your code, local packages are required in the same way :

Reviewer: ```suggestion
In your code, local packages are required in the same way:
```

Author: Thanks will do

```js
require('package-name')
```

> but global packages can run anywhere you want through shell / bash / cli

Reviewer: I find this confusing. The above example was about requiring the package not running it's binary

Author: @MylesBorins  you are right,  i can remove that line . Thanks :)

- **local packages** are installed in the directory where you run `npm install <package-name>`, and they are put in the `node_modules` folder under this directory
- **global packages** are all put in a single place in your system (exactly where depends on your setup), regardless of where you run `npm install -g <package-name>`

> In your code, local packages are required in the same way:

Reviewer: So an interesting issue here. Globally installed node_modules are not accessible by require by default. More details here

https://github.com/nodejs/node/blob/master/doc/api/modules.md#loading-from-the-global-folders

This text was ever accurate and we should likely fix that

Reviewer: ```suggestion
In your code you can only require local packages:
```
